crates/ruff_python_formatter/src/format/expr.rs

```diff
@@ -39,7 +38,7 @@ fn format_name(
     expr: &Expr,
     _id: &str,
 ) -> FormatResult<()> {
-    write!(f, [literal(Range::from_located(expr))])?;
+    write!(f, [literal(expr.into())])?;
```

Comment:
Nit: I prefer using `Range::from(expr)` for its explicitness (easier to review without type hints of an IDE)

Reply:
I guess same question applies as in #3377: do we prefer `.into()`, `Range::from`, or making `Literal` generic on `Into<Range>`?

Reply:
`Range::from` or the generic `Into<Range>` both seem reasonable to me, I defer to whatever feels more conventional.